app/models/user.rb

```diff
@@ -580,7 +528,7 @@ def check_for_username_change
   end
 
   def conditionally_resave_articles
-    Users::ResaveArticlesWorker.perform_async(id) if core_profile_details_changed? && !user.banned
```

Comment:
how did it work until now? :D

Reply:
🙃

<img width="134" alt="Screen Shot 2020-04-24 at 6 38 22 PM" src="https://user-images.githubusercontent.com/3102842/80262218-c811a680-865a-11ea-99c5-113a8dd35b00.png">